### PR TITLE
Simplify issue/PR templates

### PR DESCRIPTION
**Pull request by johnnyaug**
Also add a feature request template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -6,18 +6,13 @@ body:
   - type: markdown
     attributes:
       value: |
-        ## Hello Axolotl!
-        **Thank you for contributing to the lakeFS project.**
-        We appreciate the time you invest in filing a bug report and created this template to help make this process easier.
-        It's really important to have all the information and context, to ensure we can properly address your issue. 
-        Please use the following references to fill out the bug report.
+        Thank you for contributing to the lakeFS project.
+        For us to properly address your issue, please fill in the following fields.
```
Comment (arielshaqed) [inline .github/ISSUE_TEMPLATE/bug-report.yml:10]: Maybe even fewer words?
```suggestion
        The following fields help us properly address your issue:
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -29,45 +24,44 @@ body:
...
       required: false
   - type: input
     id: clients
     attributes:
-      label: Affected Clients
-      description: If applicable provide lakeFS client and version that was used
-      placeholder: ex. hadoop-lakefs 0.1.10, python lakefs-client 0.89.0
+      label: Affected clients
+      description: If applicable, provide the lakeFS client and version that was used
+      placeholder: Python lakeFS client 0.89.0
```
Comment (arielshaqed) [inline .github/ISSUE_TEMPLATE/bug-report.yml:51]: Maybe put in a known-_bad_ client version?  That way we know if it wasn't filled out.
```suggestion
      placeholder: Some client 0.0.1-unknown
```

**Review by arielshaqed — APPROVED** [review]
Neat-o!

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 2 above)
Comment (johnnyaug) [reply .github/ISSUE_TEMPLATE/bug-report.yml:51]: I prefer the specific example.

**Pull request merged**